rr_evgp/conf/mapping_local.yaml

```diff
@@ -1,57 +1,35 @@
 costmap:
-    update_frequency: 15.0
-    publish_frequency: 15.0
+    update_frequency: 30.0
+    publish_frequency: 30.0
     always_send_full_costmap: true
 
     rolling_window: true
-    width: 25
-    height: 25
-    resolution: 0.1
-    track_unknown_space: false
+    width: 100
+    height: 100
 
     plugins:
-        - {name: "static_global", type: "costmap_2d::StaticLayer"}
-        - {name: "obstacles", type: "costmap_2d::ObstacleLayer"}
-        - {name: "inflation", type: "costmap_2d::InflationLayer"}
-        - {name: "distance_field", type: "costmap_2d::InflationLayer"}
+        - {name: "global", type: "costmap_2d::StaticLayer"}
         - {name: "track_closing", type: "rr::TrackClosingLayer"}
+        - {name: "distance_field", type: "costmap_2d::InflationLayer"}
+
 
-    static_global:
+    global:
         map_topic: "/global_mapper/costmap/costmap"
         subscribe_to_updates: true
         trinary_costmap: true
-
-    obstacles:
-        observation_sources: laserscan
-        combination_method: 1
-        laserscan:
-            data_type: LaserScan
-            topic: /scan
-            frame_name: lidar
-            expected_update_rate: 15.0
-            observation_persistence: 0.0
-            marking: true
-            clearing: true
-            inf_is_valid: false
-            obstacle_range: 7.99
-            raytrace_range: 7.98
-
-    inflation:
-        inflation_radius: 0.3
-        cost_scaling_factor: 1.0
-        lethal_cost_threshold: 1
-
-    distance_field:
-        inflation_radius: 10.0
-        cost_scaling_factor: 0.1
-        lethal_cost_threshold: 255
+        track_unknown_space: false
 
     track_closing:
-        branch_pruning_size: 30
-        dilate_size: 3
-        min_enclosing_radius: 10
+        branch_pruning_size: 5
+        dilate_size: 10
+        min_enclosing_radius: 1
         reg_bounding_box_size: 30
-        extrapolate_distance: 20
-        max_angle_between_lines: 50
-        max_dist_between_lines: 50
+        extrapolate_distance: 1
+        max_angle_between_lines: 30
+        max_dist_between_lines: 20
         wall_output_dilation: 1
+
+    distance_field:
+        inflation_radius: 35
+        cost_scaling_factor: 0.4
+        lethal_cost_threshold: 255
```

Comment:
I don't think this is needed:
`The threshold value at which to consider a cost lethal when reading in a map from the map server.`

Reply:
Good point, just checked the source and that param is not used. idk why I included that